ui/apps/platform/src/Containers/Violations/Details/Deployment/DeploymentOverview.tsx

```diff
@@ -33,11 +27,9 @@ function DeploymentOverview({
                 desc={
                     <Link
                         to={
-                            !isFeatureFlagEnabled('ROX_PLATFORM_CVE_SPLIT')
-                                ? `${vulnerabilitiesWorkloadCvesPath}/deployments/${alertDeployment.id}`
-                                : hasPlatformWorkloadCveLink
-                                  ? `${vulnerabilitiesPlatformPath}/deployments/${alertDeployment.id}`
-                                  : `${vulnerabilitiesUserWorkloadsPath}/deployments/${alertDeployment.id}`
+                            hasPlatformWorkloadCveLink
+                                ? `${vulnerabilitiesPlatformPath}/deployments/${alertDeployment.id}`
+                                : `${vulnerabilitiesUserWorkloadsPath}/deployments/${alertDeployment.id}` // TODO or fall back to vulnerabilitiesAllImagesPath?
```

Comment:
In this case, since we have access to the `platformComponent` field I think we can navigate to the user workloads page specifically instead of the all images page.